lib/turkish_support/constants.rb

```diff
@@ -1,21 +1,54 @@
-module TurkishSupport
-  DOWNCASED_ALPHABET = 'abcçdefgğhıijklmnoöpqrsştuüvwxyz'
-  UPCASED_ALPHABET   = 'ABCÇDEFGĞHIİJKLMNOÖPQRSŞTUÜVWXYZ'
-
-  UNSUPPORTED_DOWNCASE_CHARS  = 'çğıiöşü'
-  UNSUPPORTED_UPCASE_CHARS    = 'ÇĞIİÖŞÜ'
-  ORDERED_CHARS               = UPCASED_ALPHABET + DOWNCASED_ALPHABET
+module TurkishSupportHelpers
+  ALPHA = {
+    lower:    'abcçdefgğhıijklmnoöpqrsştuüvwxyz',
+    upper:    'ABCÇDEFGĞHIİJKLMNOÖPQRSŞTUÜVWXYZ',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.